### PR TITLE
ui: Extract Sessions Page from DB Console

### PR DESCRIPTION
**Pull request by elkmaster**
ui: Extract Sessions Page from DB Console

extract session page into ui repo:
- most of the components moved without any or minimal changes
- styles converted to scss modules

Resolves: #57296

Release note (ui): none

### CODE REVIEW

**Comment by elkmaster** [comment]
corresponding pr for admin-ui https://github.com/cockroachdb/cockroach/pull/58034

File: packages/admin-ui-components/src/tooltip2/tooltip.tsx [hunk 1]
```diff
@@ -9,19 +9,32 @@ import styles from "./tooltip.module.scss";
...
 export const Tooltip = (props: TooltipProps & AntTooltipProps) => {
-  const { children, theme, overlayClassName } = props;
+  const { children, theme, overlayClassName, tableTitle } = props;
   const classes = cx(
     "tooltip-overlay",
     `crl-tooltip--theme-${theme}`,
     overlayClassName,
   );
+
+  const title = tableTitle ? (
```
Comment (dhartunian) [inline packages/admin-ui-components/src/tooltip2/tooltip.tsx:25]: I don't see this in the source of `pkg/ui`. Why did we need to add it?

File: packages/admin-ui-components/src/statementsTable/statementsTableContent.tsx [hunk 2]
```diff
@@ -276,19 +276,35 @@ export const StatementTableCell = {
...
   app: string;
   implicitTxn: boolean;
   search: string;
-}) => {
+  anonStatement?: string;
+}
+
+// StatementLinkTarget returns the link to the relevant statement page, given
+// the input statement details.
+export const StatementLinkTarget = (props: StatementLinkProps) => {
```
Comment (dhartunian) [inline packages/admin-ui-components/src/statementsTable/statementsTableContent.tsx:289]: Would this introduce problems in CC if we need to route to a different URL set?

**Review by dhartunian — CHANGES_REQUESTED** [review]
Just minor questions, otherwise looks good.

File: packages/admin-ui-components/src/tooltip2/tooltip.tsx (hunk 1 above)
Comment (elkmaster) [reply packages/admin-ui-components/src/tooltip2/tooltip.tsx:25]: we have different style for table title tooltip. to achieve that earlier we added <div/> with corresponding class(https://github.com/cockroachdb/cockroach/blob/master/pkg/ui/src/views/sessions/sessionsTableContent.tsx), but to remove duplications I moved this on tooltip component level.

File: packages/admin-ui-components/src/statementsTable/statementsTableContent.tsx (hunk 2 above)
Comment (elkmaster) [reply packages/admin-ui-components/src/statementsTable/statementsTableContent.tsx:289]: Looks like it should not. Current change is just moving out basically the same url forming logic to separate function. But I will double check it when will be working on CC integration/

**Review by dhartunian — APPROVED** [review]

**Pull request merged**